hack/releasechangelog.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
```

Comment:
I'm wondering if we pass in the gh cli path, version etc as args to the script instead?

Reply:
Do we need to run envsubst on the actual script file? It might be better to pass in arguments to the script file instead?

Reply:
This way the script can be run independently as well

Reply:
I have made these changes 👍